framework/base/src/contract_base/wrappers/storage_raw_wrapper.rs

```diff
@@ -86,4 +87,18 @@ where
         let key: StorageKey<A> = storage_key.into();
         storage_set(key.as_ref(), value);
     }
+
+    pub fn token_has_transfer_role(&self, token_identifier: TokenIdentifier<A>) -> bool {
```

Comment:
I would move it to the BlockchaiWrapper, there are more ESDT-related methods there.